vm/src/stdlib/builtins.rs

```diff
@@ -87,8 +89,8 @@ mod builtins {
     }
 
     #[pyfunction]
-    fn chr(i: u32, vm: &VirtualMachine) -> PyResult<String> {
-        match std::char::from_u32(i) {
+    fn chr(i: BigInt, vm: &VirtualMachine) -> PyResult<String> {
```

Comment:
A `PyInt` probably better matches what CPython does.

Reply:
Oh, I didn't recognize this is added. `TryFromObject for BigInt` cannot avoid clone. So it will mislead cost. `PyInt` must be used.
I will remove TryFromObject for BigInt.

Reply:
#3224

Reply:
Done! Changed to a PyIntRef